src/codegen.cpp

```diff
@@ -4422,6 +4422,9 @@ static std::unique_ptr<Module> emit_function(
         // TODO: Fix when moving to new LLVM version
         #if JL_LLVM_VERSION < 30400
         dbuilder.createCompileUnit(0x01, filename, ".", "julia", true, "", 0);
+        #elif JL_LLVM_VERSION >= 40000
```

Comment:
we're a little inconsistent about whether or not we indent ifdefs - most of the time it looks like we don't though